controllers/k8ssandra/schemas.go

```diff
@@ -86,6 +86,7 @@ func (r *K8ssandraClusterReconciler) checkInitialSystemReplication(
 
 	replication := make(map[string]int)
 	if val := annotations.GetAnnotation(kc, api.InitialSystemReplicationAnnotation); val != "" {
+		// TODO if a value exists, do we need to filter it to only keep the first DC?
```

Comment:
I'm not sure about this, but I would guess that if we already have the annotation, then there is at least one DC so we  won't need the value anymore.

Reply:
I guess we'd need to test this. The annotation is only important when the cluster is bootstrapped indeed, and I think the operator creates that annotation, right?
Now, can a user override it and put something that would break it?
If so, you'll need a check and fail the reconcile.

Reply:
Yes, the operator creates the annotation. Not only is it only used for bootstrapping, but with my changes it's now only applied to the first DC. The code already handles bad input.
So all things considered I think we're good, I'll remove the TODO.